.pre-commit-config.yaml

```diff
@@ -15,7 +15,8 @@ repos:
     hooks:
       - id: ruff
         args: [--fix]
-        exclude: ^.*/Tiltfile$
+        files: ^.*Tiltfile|^.*\.py$
+        types_or: [text]
```

Comment:
Can be updated to `types_or: [python, starlark]` (without further `files` definition) once this gets accepted https://github.com/pre-commit/identify/pull/494 and released.

Reply:
done